src/sentry/lang/native/symbolicator.py

```diff
@@ -217,6 +223,29 @@ def process_js(self, stacktraces, modules, release, dist, apply_source_context=T
 
         return self._process("symbolicate_js_stacktraces", "symbolicate-js", json=json)
 
+    def process_jvm(
+        self,
+        exceptions,
+        stacktraces,
+        modules,
+        release_package,
+        apply_source_context=True,
+    ):
+        source = get_internal_artifact_lookup_source(self.project)
```

Comment:
this shouldn’t be the `artifact_lookup` source, but rather the "native" symbol source.